pyproject.toml

```diff
@@ -23,13 +23,14 @@ exclude = ["notebooks/*"]
 readme = "README.rst"
 
 [tool.poetry.dependencies]
-python = ">= 3.6.1"
+python = ">= 3.8"
 jupyter = "^1.0.0"
 pandas = "^1.0.0"
+ipytablewidgets = "^0.2.3"
 
 [tool.poetry.dev-dependencies]
 pytest = "^6.2"
-altair = "^4.0.1"
+altair = "^4.2.0"
```

Comment:
Wait, so Altair is a Dev dependency. The note in the notebook is not correct then.